DOMManipulation/DOMManipulation.js

```diff
@@ -0,0 +1,17 @@
+const main = document.querySelector("main");
+const button = document.createElement("button");
+button.textContent = "Click me!";
+
+main.appendChild(button);
+
+button.addEventListener("click",addHtmlElement)
+
+function addHtmlElement(){
+    const heading1 = document.createElement("h1");
+    heading1.innerHTML = "Javascript test";
+    const body = document.querySelector("body");
+    body.style.backgroundColor = "yellow";
+    button.disabled = true;
+    
+    main.append(heading1);
+}
```

Comment:
Well done! Nothing really to correct here. Just a very small piece of advice (and this is really just advice, not correction) is that it would be a better idea to use `querySelector("main")` within the `addHtmlElement` function, just so you have it as a local variable in the function, and you do not have to read it from the global closure. But other than that it is perfect! ⚡